app/views/layouts/application.html.erb

```diff
@@ -1,37 +1,38 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <meta charset="utf-8">
   <title>PekNext</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': true %>
   <%= csrf_meta_tags %>
 </head>
+
 <body class="uk-height-1-1">
 
-<%= render 'navbar' unless controller_name == 'registration' %>
+  <%= render 'navbar' unless (controller_name == 'registration') || (controller_name == 'login') %>
 
-<% if flash.alert %>
-  <div class="uk-alert uk-alert-danger uk-container-center uk-width-2-3 uk-margin-top">
-    <p>
-      <%= flash.alert %>
-    </p>
-  </div>
-<% end %>
+  <% if flash.alert %>
+    <script>
+      UIkit.notify("<%= flash.alert %>", {status:'danger'});
```

Comment:
It is mainly the same notification that was before, just this is nicer usage of UIkit. You can try these notifications [here](https://getuikit.com/v2/docs/notify.html).